genesis/parsing/accountsParser_test.go

```diff
@@ -74,15 +74,17 @@ func createDelegatedInitialAccount(address string, delegatedBytes []byte, delega
 func TestNewAccountsParser_NilEntireBalanceShouldErr(t *testing.T) {
 	t.Parallel()
 
-	ap, err := parsing.NewAccountsParser(
-		"./testdata/genesis_ok.json",
-		nil,
-		"",
-		createMockHexPubkeyConverter(),
-		&mock.KeyGeneratorStub{},
-		&mock.HasherMock{},
-		&mock.MarshalizerMock{},
-	)
+	args := genesis.AccountsParserArgs{
```

Comment:
usually we create an unexported function in the test file, something like `createMockAccountsParserArgs` and return a populated argument struct that will work as it is in the xxx_ShouldWork test.
On the other tests, like in the TestNewAccountsParser_NilEntireBalanceShouldErr we could have something like:
```GO
args := createMockAccountsParserArgs()
args.EntireSupply = nil
////.......
```
In this way the duplicate code can be removed :)

Reply:
added additional function :+1: